recipe/meta.yaml

```
@@ -37,33 +31,35 @@ requirements:
   build:
     - python
     # run-dep as a workaround for now; upstream package still imports from distutils
-    - setuptools <74  # [py>311]
+    - setuptools <74
+    - {{ compiler('c') }}
     - {{ posix }}bison
     - {{ posix }}flex
     - {{ posix }}patch
     - {{ posix }}sed
     - meson >=0.60.0
     - ninja-base
     - pkg-config
-    - {{ compiler('c') }}
   host:
-    - pkg-config
     - python
     # X11/XCB dependencies through cairo.
     - cairo {{ cairo }}
-    - glib {{ glib }}
+    - glib >=2.82,<2.85
```

```suggestion
    - glib >=2.82.0,<2.85.0
```
Where does the upper bound come from? I found only this https://gitlab.gnome.org/GNOME/gobject-introspection/-/blob/1.84.0/meson.build?ref_type=tags#L156